projects/plugins/boost/changelog/update-boost-add-tracks-events

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: added
+
+Boost: add Tracks events to clickable events on settings page
```

Comment:
The change is within the Boost plugin. No need to specify that. Specify the aspect of Boost the changes affect.

Reply:
Thanks, done.